src/com/crossbowffs/quotelock/modules/wikiquote/WikiquoteQuoteModule.java

```diff
@@ -0,0 +1,70 @@
+package com.crossbowffs.quotelock.modules.wikiquote;
+
+import android.content.ComponentName;
+import android.content.Context;
+
+import com.crossbowffs.quotelock.R;
+import com.crossbowffs.quotelock.api.QuoteData;
+import com.crossbowffs.quotelock.api.QuoteModule;
+import com.crossbowffs.quotelock.modules.goodreadsqotd.GoodreadsQuoteModule;
+import com.crossbowffs.quotelock.utils.IOUtils;
+import com.crossbowffs.quotelock.utils.Xlog;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class WikiquoteQuoteModule implements QuoteModule {
+    private static final String TAG = WikiquoteQuoteModule.class.getSimpleName();
+
+    @Override
+    public String getDisplayName(Context context) {
+        return context.getString(R.string.module_wikiquote_name);
+    }
+
+    @Override
+    public ComponentName getConfigActivity(Context context) {
+        return null;
+    }
+
+    @Override
+    public QuoteData getQuote(Context context) throws IOException {
+
+        String quotePage = IOUtils.downloadString("https://zh.m.wikiquote.org/zh-cn/Wikiquote:%E9%A6%96%E9%A1%B5");
+        Xlog.i(TAG, "Quote page 提取：%s", quotePage);
+
+        String regexQuoteAllText = "(?<=<td>).*?(?=</td>)";
+        String regexRemoveFormat = "<.*?>";
+        Matcher quoteAllTextMatcher = Pattern.compile(regexQuoteAllText).matcher(quotePage);
+        boolean quoteAllTextStatus = quoteAllTextMatcher.find();
```

Comment:
I think this needs a success check (what if the regex does not match?) Just `return null` if it does not match